gnovm/cmd/gno/build.go

```diff
@@ -7,17 +7,16 @@ import (
 	"os"
 
 	gno "github.com/gnolang/gno/gnovm/pkg/gnolang"
+	"github.com/gnolang/gno/gnovm/pkg/gnoutil"
 	"github.com/gnolang/gno/tm2/pkg/commands"
 )
 
 type buildCfg struct {
-	verbose  bool
-	goBinary string
```

Comment:
removed the option to specify a gno file as:

- this is inconsistent with the rest of `guessRootDir`, which just use "go" as an argument.
- it is already fairly straightforward to change this in a script; just change your `PATH`.